bots/images/scripts/fedora-atomic.bootstrap

```diff
@@ -10,4 +10,4 @@ BASE=$(dirname $0)
 # The Fedora URLs have the version twice in the name. for example:
 # https://dl.fedoraproject.org/pub/alt/atomic/stable/Fedora-Atomic-25-20160921.0/CloudImages/x86_64/images/Fedora-Cloud-Base-25-20160921.0.x86_64.qcow2
 $BASE/atomic.bootstrap "$1" "$url" \
-    "Fedora-Atomic-25-[0-9\.]+" "CloudImages" "x86_64" "images" "Fedora-Atomic-25-[0-9\.]+.x86_64.qcow2"
+    "Fedora-Atomic-[-0-9\.]+" "CloudImages" "x86_64" "images" "Fedora-Atomic-[-0-9\.]+.x86_64.qcow2"
```

Comment:
If we are trying to have the image automatically jump between versions then we might need to try to figure out the build OS automatically in image-prepare as well as the build image needs to match the atomic version. Otherwise we'll ended trying to install packages for the wrong OS.